tests/gnrc_dhcpv6_client/tests-as-root/01-run.py

```diff
@@ -15,6 +15,10 @@ def testfunc(child):
     child.expect(r"inet6 addr:\sfe80:[0-9a-f:]+\s+scope: link")
     child.expect(r"Iface\s+\d+")
     child.expect(r"inet6 addr:\s+fe80:[0-9a-f:]+\s+scope: link")
+    child.expect(r"inet6 addr:\s+(?P<ia_na_addr>[0-9a-f:]+)\s+scope: global")
+    ia_na_addr = child.match.group("ia_na_addr")
+    assert ia_na_addr.startswith(
+        "2001:db8:1::"), "IA_NA address has not been assigned correctly"
```

Comment:
Maybe use a variable here instead of the literal `"2001:db8:1::"`?

Reply:
I added a variable in 28de178a46d3d2b4e54d24bc9259447c9d638b13